tests/unit/schema/test_api.py

```diff
@@ -1595,3 +1596,151 @@ def test_http_options(fs):
                 method='get', uri='/v3/{name=projects/*/locations/*/operations/*}', body=None),
             wrappers.HttpRule(method='get', uri='/v3/{name=/locations/*/operations/*}', body=None)]
     }
+
+
+def generate_basic_diregapic_setup():
```

Comment:
WDYT about using `extended_operations` as a suffix/prefix here? Because seems like this is what the feature is about and "diregapic" is technically an iternal term (unlike extended_operations which were published by this name in googleapis).

Reply:
That seems reasonable. I'll make the change throughout.